pkg/descheduler/evictions/evictions.go

```diff
@@ -113,6 +116,17 @@ type EvictOptions struct {
 // EvictPod evicts a pod while exercising eviction limits.
 // Returns true when the pod is evicted on the server side.
 func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, opts EvictOptions) bool {
+	var err error
+	ctx, evictSpan, evictCloser := tracing.StartSpanWithOptions(ctx, tracing.EvictionOption, []trace.TracerOption{}, "EvictPod", []attribute.KeyValue{
+		{Key: "pod-name", Value: attribute.StringValue(pod.GetName())},
+		{Key: "pod-namespace", Value: attribute.StringValue(pod.GetNamespace())},
```

Comment:
we could pass these simpler with the [attribute](https://pkg.go.dev/go.opentelemetry.io/otel/attribute#String) package helpers (example in k8s: https://github.com/kubernetes/kubernetes/blob/d29e3bd7aa4978f10a01b9dfdebb2a647634f8fe/staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go#L367-L371)

Reply:
This brings up an interesting thing. Instead of us wrting some of these handlers ourself. should we just use teh `component-base/tracing` directly for all the use cases ?

Reply:
But then again, they still don't support Certs and others on the trace endpoints
https://github.com/kubernetes/component-base/blob/ae9be4dda9dabe6503eff5dd6162f48247880eea/tracing/api/v1/types.go#L20